packages/taquito-utils/src/errors.ts

```diff
@@ -58,7 +58,7 @@ export class InvalidProtocolHashError extends ValidationError {
  *  @category Error
  *  @description General error that indicates a failure when trying to convert data from one type to another
  */
-export class ValueConversionError extends ValidationError {
+export class ValueConversionError extends InternalValidationError {
```

Comment:
should this be ParameterValidationError? 
its used only in the hex2buf helper function 
it may be used internally however the function alone should be by parameter

Reply:
good catch, will update now